ocs_ci/helpers/helpers.py

```diff
@@ -1309,13 +1309,14 @@ def get_start_creation_time(interface, pvc_name):
     return datetime.datetime.strptime(start, format)
 
 
-def get_end_creation_time(interface, pvc_name):
+def get_end_creation_time(interface, pvc_name, last_end_time=False):
```

Comment:
@Avilir - of course your logic is correct, it's true to take the last timestamp of the last record in logs for the tests I know. 
However, I did not want to change the function in helpers, since there might be other tests that still need the "first appearance in log" timestamp. 
Are we sure that there are no such tests? 
If yes, I can change the implementation in helpers so that ALWAYS the last timestamp will be taken. 
But in case we are not sure, it's better to leave it this way for now.

Reply:
Need to check all those functions in helpers, need to remove code duplication https://github.com/red-hat-storage/ocs-ci/issues/3403

Reply:
@Avilir , @keesturam - I've fixed the functions in helpers.py so that the latest timestamp will be taken. 
@ramkiperiy - Thank you for listing all those functions. Let's make sure that while fixing PR#3403 the correct (latest) relevant timestamp will be taken to measurement.